Util/Xamarin.Build.Download/source/Xamarin.Build.Download.Tests/Helpers/TestsBase.cs

```diff
@@ -28,8 +27,10 @@ public void SetUp ()
 			Environment.CurrentDirectory = tempDir;
 		}
 
-		[OneTimeTearDown]
-		public void TearDown ()
+        public bool IsWindows
```

Comment:
I know this is just text, but I still hope the files will have either tabs or spaces.